static/includes/Reference/SCALEServicesSNMPFields.md.part

```diff
@@ -0,0 +1,26 @@
+**General Options**
+
+| Name | Description |
+|------|-------------|
+| Location | Enter the location of the system. |
+| Contact | E-mail address that receives SNMP service messages. |
+| Community | Change from *public* to increase system security. Can only contain alphanumeric characters, underscores (`_`), dashes (`-`), periods (`.`), and spaces. This can be left empty for SNMPv3 networks. |
```

Comment:
```suggestion
| Community | Change from **public** to increase system security. Can only contain alphanumeric characters, underscores (`_`), dashes (`-`), periods (`.`), and spaces. This can be left empty for SNMPv3 networks. |
```